src/components/ComposeTextPanel/index.js

```diff
@@ -111,7 +111,7 @@ class ComposeTextPanel extends Component {
       {
         id: '1',
         level: 'warning',
-        message: messageSenderMessages.senderNumberInvalids,
+        message: messageSenderMessages.senderNumbersInvalids,
```

Comment:
@caffffe Please rename senderNumberInvalids and senderNumbersInvalids to senderNumberInvalid and senderNumbersInvalid respectively. This is not urgent, but is more grammatically correct.